Feature/past tenants

eashpie7u7 opened the pull request:
### Summary
This pr adds a new PastTenantsList component that visually renders a card contains the profile photo, agreement date, specialty and rating of a past tenant that rented the place. It also included a Star Rating component that is used 

### Changes

- Add PastTenantList component and documentation
- Added PastTenantCard 
- Added StarRating logic that is used in the cards

<img width="1512" alt="Screenshot 2025-04-12 at 7 40 55 p m" src="https://github.com/user-attachments/assets/e1ddb53e-896d-4788-9532-4e65701dd194" />

CarlosP1806 commented on src/components/PastTenants/PastTenantCard.tsx:
```diff
@@ -0,0 +1,58 @@
+    <div className="border rounded-lg shadow-sm p-4 bg-white flex items-center justify-between">
+      <div className="flex items-center gap-3">
+        <Image
+          src={profile_pic}
```
Creo que sería mejor de una vez pasar la url a la profile_pic como prop en lugar de hardcodear un valor

CarlosP1806 commented on src/components/PastTenants/PastTenantList.tsx:
```diff
@@ -0,0 +1,46 @@
+import { FC } from "react";
+import PastTenantCard, { PastTenant } from "./PastTenantCard";
+
+const pastTenants: PastTenant[] = [
```
Esto esta bien para probar, pero bórralo del código final porfa

eashpie7u7 commented:
### Summary
This pr adds a new NotificationsView component that visually renders a card contains a notification. It also included a Dismiss button to close the notification

### Changes
Added NotificationView component
Added NotificationCard component 

<img width="1512" alt="Screenshot 2025-04-13 at 10 25 21 a m" src="https://github.com/user-attachments/assets/cd52c9c3-45d7-403f-a286-62ea7f949c32" />

CarlosP1806 reviewed (approved):
Ok!
